### PR TITLE
feat: add option labels

### PR DESCRIPTION
**Pull request by xTrekStorex**
This adds the option `labels` to the reporter, which implements the external_label behavior of prometheus remote write.
By default it's undefined and sets an empty array in prometheus-remote-write options, so there are no breaking changes.

Implements https://github.com/vitalics/playwright-prometheus-remote-write-reporter/issues/9

### CODE REVIEW

**Comment by vitalics** [comment]
please make rebase(or merge) from current master

**Comment by xTrekStorex** [comment]
> please make rebase(or merge) from current master

done

**Comment by xTrekStorex** [comment]
regarding the build errors: I am no nodejs dev and never touched typescript. so I have no idea how to properly define an array of key value pairs as type. all the docs I read suggested `Record<string, string>[]` but that seems to be invalid.

the lib expects an array of key value pairs:
```
labels: Array.isArray(t.labels)
  ? [t.labels, ...(kv(options?.labels) || [])]
  : kv({
      ...options?.labels,
      ...t.labels,
    }),
```

**Comment by xTrekStorex** [comment]
fixed and build tested locally

File: README.md [hunk 1]
```diff
@@ -54,8 +54,10 @@ export default defineConfig({
 | prefix        | Custom metric prefix name                                 | `pw_`                                |
 | auth.username | Basic auth. username                                      | undefined                            |
 | auth.password | Basic auth. password                                      | undefined                            |
+| labels        | Ext. labels for all metrics. E.g. `[{label1: 'value1'}]`  | undefined                            |
```
Comment (vitalics) [inline README.md:57]: please update the example. It should be an object

File: src/index.ts [hunk 2]
```diff
@@ -46,6 +46,12 @@ export type PrometheusOptions = {
   };
   /** @default 'pw_' */
   prefix?: string;
+  /**
+   * Additional labels to apply to each timeseries.
+   * @example
+   * [{ instance: "hostname" }]
```
Comment (vitalics) [inline src/index.ts:52]: example and type missmatched

**Review by vitalics — CHANGES_REQUESTED** [review]

**Review by vitalics — APPROVED** [review]
LGTM!

**Pull request merged**